### ISSUE TITLE
add profile to spdm-toc-map

### ISSUE THREAD

**Issue by nedmsmith** [TrustedComputingGroup/concise-evidence#25]
The `spdm-toc-map` should contain a profile.
In the case where indirect measurements are to be mapped to direct measurements, the lead Attester may have several mapping conventions available. A profile would allow the lead Attester to select a mapping convention that is appropriate for the deployment context. 
```
spdm-toc-map = {
  &(tagged-evidence: 0) => [ + $tagged-evidence-type-choice]
  ? &(rim-locators: 1) =>  [ + corim-locator-map ] ;see corim
  ? &(profile: 2) => profile-type-choice ; see corim
  * $$spdm-toc-map-extension
}
```
The `profile` code point should be added and use the `profile-type-choice` definition from corim.

**Comment by nedmsmith** [comment]
Fixed by PR #26

**Comment by nedmsmith** [comment]
fixed by PR #26
